src/ReactZenDeskChat/ReactZenDeskChat.js

```diff
@@ -86,7 +86,6 @@ export default class ZenDeskChat extends Component {
 				title && this.setTitleText(title);
 				preChatGreeting && this.setPreChatGreeting(preChatGreeting);
 				badgeText && this.setBadgeText(badgeText);
-				delete window.$zopim;
```

Comment:
This was preventing zopim from being available to change the language at runtime.